### DISCUSSION TITLE
Invoking external "build environment" e.g. make

### DISCUSSION THREAD

**Discussion by MartinSchwarzbauer** [mesonbuild/meson#12941] [Q&A]
Hi *!

We have a third party component which uses a shell script (to set variables) which invokes a makefile!
How can i call this make file (at best case during compile?)

But how to path the whole CFLAGS / CC / and so on from meson?

**Comment by eli-schwartz** [comment]
Does it use a configure script (not necessarily a GNU autotools one, but at least a configure script that accepts arguments like --prefix --libdir etc.)?

https://mesonbuild.com/External-Project-module.html

**Comment by MartinSchwarzbauer** [comment]
No it's not using configure! 

A shell script sets various environment variables and then invokes make -f ....
